scripts/moon-surface.lua

```diff
@@ -112,11 +112,41 @@ local function on_configuration_changed()
       map_gen_settings.autoplace_settings.entity.settings["ll-astrocrystals"] = default_resource_controls
       luna.map_gen_settings = map_gen_settings
     end
+
+    if not global.migrations.luna_cliffs_made_indestructable then
+      global.migrations.luna_cliffs_made_indestructable = true
+
+      local cliffs = luna.find_entities_filtered{
+        name = "ll-luna-cliff",
+      }
+
+      for _, cliff in ipairs(cliffs) do
+        cliff.destructible = false
+      end
+    end
+  end
+end
+
+local function on_chunk_generated(event)
+  if event.surface.name ~= 'luna' then return end
+
+  local cliffs = event.surface.find_entities_filtered{
```

Comment:
Probably better to do a script-trigger on_created, for future reference

Reply:
how would that even work? i've not heard of world gen being able to trigger such events?